lib/src/http_session.dart

```diff
@@ -139,6 +120,17 @@ class HttpSession implements IOClient {
     }));
   }
 
+  @override
+  Future<http.Response> delete(
+    Uri url, {
+    Map<String, String>? headers,
+    Object? body,
+    Encoding? encoding,
+  }) async {
+    return _sendRequest("POST", url, maxRedirects,
```

Comment:
I think this should be "DELETE".

Reply:
Oh wow I'm dumb yes it does need to be DELETE

Reply:
I'm fixing this also I'm adding tests to make sure nothing like this gets missed in the future

Reply:
Done